wfl/generate/md/__init__.py

```diff
@@ -149,20 +154,28 @@ def _sample_autopara_wrappable(atoms, calculator, steps, dt, temperature=None, t
             # one stage, simple
             all_stage_kwargs = [stage_kwargs.copy()]
             all_run_kwargs = [ {'steps': steps} ]
+
         else:
             # NVT or NPT
             all_stage_kwargs = []
             all_run_kwargs = []
 
-            stage_kwargs['taut'] = temperature_tau * fs
+#            stage_kwargs['taut'] = temperature_tau * fs
 
             if pressure is not None:
                 md_constructor = NPTBerendsen
                 stage_kwargs['pressure_au'] = pressure
                 stage_kwargs['compressibility_au'] = compressibility
                 stage_kwargs['taup'] = temperature_tau * fs * 3 if pressure_tau is None else pressure_tau * fs
             else:
-                md_constructor = NVTBerendsen
+                if integrator == "NVTBerendsen":
+                    md_constructor = NVTBerendsen
+                    stage_kwargs['taut'] = temperature_tau * fs
+
+                elif integrator == "Langevin":
+
+                    md_constructor = Langevin
+                    stage_kwargs["friction"] = 1 / temperature_tau
```

Comment:
Any reason why you aren't doing the fs unit conversion here as well (although since ASE's native units are fs, it doesn't actually do anything)?